.github/workflows/mac.yml

```diff
@@ -45,8 +45,8 @@ jobs:
       id: openvino_download
       with:
         platform: macos_12_6
-        commit_packages_to_provide: wheels
-        revision: latest_nightly
+        commit_packages_to_provide: wheels,openvino_js_package.tar.gz
+        revision: latest_available_commit
```

Comment:
I'm not sure that latest available commit is available on macOS since we don't have precommit on macOS in OpenVINO GHA infra.

Reply:
We introduced it here - https://github.com/openvinotoolkit/openvino/pull/29086

Reply:
but it would not be available for release package, @Retribution98, we would need to think about that too